.eslintrc.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
+    extends: [
+      "plugin:prettier/recommended"
```

Comment:
> - Integrated Prettier with ESLint to avoid conflicts

Is this the related commit?

Reply:
This will disables the conflicting ESLint rules.